src/__tests__/featureflags.js

```diff
@@ -191,6 +191,53 @@ describe('featureflags', () => {
             })
         })
     })
+
+    describe('when subsequent decide calls return partial results', () => {
```

Comment:
@neilkakkar I tested when existing data already exists and subsequent calls are partial. wdyt